### PR TITLE
RA-1846:Update Metadata Packages for ReferenceMetadata module

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RA-1846
cc @dkayiwa  @ibacher  @mozzy11

### CODE REVIEW

**Comment by sherrif10** [comment]
here is the same log https://pastebin.com/UqCkBYsu we can start from debugging from here

**Comment by mozzy11** [comment]
@sherrif10   , can you also first remove remove the numeric concepts and place them in he appropriate packages ?

**Comment by sherrif10** [comment]
cc @mozzy11 @dkayiwa  could you clarify this Pr about removing form-text, i wanted this to be merged then i can push another work on the same pr just to keep trach

**Comment by mozzy11** [comment]
Thanks @sherrif10 

LGTM cc @dkayiwa

File: api/src/main/resources/Reference_Application_Diagnoses-13.xml [hunk 1]
```diff
@@ -320,7 +320,7 @@
...
+<concept_description concept_description_id="9762" concept_id="132636" description="A syndrome produced by severe protein deficiency, characterized by retarded growth, changes in skin and hair pigment, edema, and pathologic changes in the liver, including fatty infiltration, necrosis, and fibrosis. The word is a local name in Gold Coast, Africa, meaning displaced child. Although first reported from Africa, kwashiorkor is now known throughout the world, but mainly in the tropics and subtropics. It is considered to be related to marasmus. (From Dorland, 27th ed)" locale="en" creator="1" date_created="2007-10-18 04:28:24.0" changed_by="1" uuid="9762FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"/>
```
Comment (dkayiwa) [inline api/src/main/resources/Reference_Application_Diagnoses-13.xml]: What is the purpose of this change?

File: api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml [hunk 2]
```diff
@@ -1214,7 +1214,7 @@
...
 <concept_reference_source concept_source_id="27" name="NDF-RT NUI" description="Unique numeric indicator from NDF-RT drug information database" creator="1" date_created="2014-02-07 22:15:18.0" retired="false" retired_by="1" uuid="27ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
-<concept_reference_source concept_source_id="28" name="FDA Route of Administration" description="Description. This standard provides for all routes of administration for drugs. Source. COMIS Reference table (which is used by the Drug Product Reference File to generate Approved Drug Products with Therapeutic Equivalence Evaluations (a.k.a. "The Orange Book")), and the Drug Registration and Listing Database, Office of Epidemiology and Biosurveillance Database. Also, the names (but not the definitions or the numeric codes) that are represented in blue have been harmonized with the E2B route of administration terms for the International Conference on Harmonization (ICH)." creator="1" date_created="2015-01-16 18:06:56.0" retired="false" retired_by="1" uuid="28ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
```
Comment (dkayiwa) [inline api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml]: What is the purpose of this change?

File: api/src/main/resources/Reference_Application_Concepts-26.xml [hunk 3]
```diff
@@ -117,16 +117,6 @@
 <concept concept_id="162822" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-01-06 22:33:22.0" version="" changed_by="1" date_changed="2015-01-06 22:33:29.0" retired_by="1" uuid="162822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
 <concept concept_id="163139" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2015-12-13 15:00:18.0" version="" changed_by="1" date_changed="2015-12-13 15:02:03.0" retired_by="1" uuid="163139AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
 <concept concept_id="165095" retired="false" datatype_id="3" class_id="7" is_set="false" creator="1" date_created="2018-04-18 14:34:20.0" version="" changed_by="1" date_changed="2018-04-18 14:34:30.0" retired_by="1" uuid="165095AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept_numeric concept_id="1343" units="cm" allow_decimal="true"/>
-<concept_numeric concept_id="5085" hi_absolute="250.0" low_absolute="0.0" units="mmHg" allow_decimal="false"/>
```
Comment (dkayiwa) [inline api/src/main/resources/Reference_Application_Concepts-26.xml]: Did you just decide to throw away these numerics?

File: api/src/main/resources/Reference_Application_Diagnoses-13.xml (hunk 1 above)
Comment (sherrif10) [reply api/src/main/resources/Reference_Application_Diagnoses-13.xml]: With this change i was making it a full  single string by removing double quotes on "display child" which was a quick fix

File: api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml (hunk 2 above)
Comment (sherrif10) [reply api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml]: @dkayiwa: same here its format after import wasn't accurate, so i hard to make if full  single string

File: api/src/main/resources/Reference_Application_Concepts-26.xml (hunk 3 above)
Comment (sherrif10) [reply api/src/main/resources/Reference_Application_Concepts-26.xml]: @dkayiwa  Nope , i rather updated them in a separate dataset file called reference_application_Numeric_concepts.25.xml. so there was a need to remove them from that file thanks

File: api/src/main/resources/Reference_Application_Diagnoses-13.xml (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/resources/Reference_Application_Diagnoses-13.xml]: What error was it giving?

File: api/src/main/resources/Reference_Application_Concepts-26.xml (hunk 3 above)
Comment (dkayiwa) [reply api/src/main/resources/Reference_Application_Concepts-26.xml]: In your commit, i do not see a file with that name.

Comment (sherrif10) [reply api/src/main/resources/Reference_Application_Concepts-26.xml]: You can check in Reference_application_Numeric_Concepts-26-2.xml file,  it only contains numeric concepts extracted from Reference_application_concepts-26.xml. thanks

File: api/src/main/resources/Reference_Application_Diagnoses-13.xml (hunk 1 above)
Comment (sherrif10) [reply api/src/main/resources/Reference_Application_Diagnoses-13.xml]: Element type "concept_description" must be followed by either attribute specifications, ">" or "/>" element unterminated thats was the error it was giving

File: api/src/main/resources/Reference_Application_Concepts-26.xml (hunk 3 above)
Comment (dkayiwa) [reply api/src/main/resources/Reference_Application_Concepts-26.xml]: But i still do not see the file you said above named `reference_application_Numeric_concepts.25.xml`

File: api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml [hunk 4]
```diff
@@ -10,4 +10,4 @@
 <concept_numeric concept_id="5092" hi_absolute="100.0" low_absolute="0.0" units="%" allow_decimal="false"/>
 <concept_numeric concept_id="5242" hi_absolute="999.0" hi_critical="120.0" hi_normal="70.0" low_absolute="0.0" units="" allow_decimal="false"/>
```
Comment (sherrif10) [inline api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml:11]: cc @dkayiwa  this is the file i meant: So i think with current numeric concepts  there were no changes with latter concepts, thats why it showed as if i didn't update them thanks
